selftests/unit/test_utils_process.py

```diff
@@ -321,6 +336,36 @@ def test_run_unicode_output(self):
         self.assertEqual(result.stdout, encoded_text)
         self.assertEqual(result.stdout_text, text)
 
+    def test_run_with_timeout_ugly_cmd(self):
+        content = (REFUSE_TO_DIE)
```

Comment:
I don't get the reason for this line, it just adds a variable declaration that does nothing as far as I can tell.  If it is for aesthetics reasons, the following looks just as good to me:

```
        with script.TemporaryScript("refuse_to_die", REFUSE_TO_DIE) as exe:
```

Reply:
That's just a left-over, sorry.